src/services/homework/hooks/submissions.js

```diff
@@ -274,26 +260,34 @@ const noDuplicateSubmissionForTeamMembers = (context) => {
 		// check if a teamMember submitted a solution on his own => display names
 		let newTeamMembers = context.data.teamMembers;
 		if (context.data.submission) {
-			newTeamMembers = newTeamMembers.filter((teamMember) => !context.data.submission.teamMembers.includes(teamMember.toString()));
+			newTeamMembers = newTeamMembers.filter(
+				(teamMember) => !context.data.submission.teamMembers.includes(teamMember.toString())
+			);
 		}
 
 		const submissionsForTeamMembers = context.data.submissions.filter((submission) => {
 			for (let i = 0; i < newTeamMembers.length; i += 1) {
 				const teamMember = newTeamMembers[i].toString();
-				if (submission.teamMembers.includes(teamMember)
-						|| (((submission.studentId || {})._id || {}).toString() == teamMember)
+				if (
+					submission.teamMembers.includes(teamMember) ||
+					((submission.studentId || {})._id || {}).toString() == teamMember
```

Comment:
Codacy found an issue: [Expected '===' and instead saw '=='.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6045788)